package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "concord-react-components",
```

Comment:
FYI, several other CC npm packages are published under the `@concord-consortium` scope. You might consider doing the same here, i.e. `@concord-consortium/react-components`.

Reply:
Thanks -- I wasn't sure about that, but I see basically all of our packages are done that way.  So I updated it in  19431a8